LLVM clang [3.9.0,9.0.0) (and so likely also Xcode clang <11.4)
defined the _mm256_maskz_cvtps_ph intrinsic as 2-argument.
Use the equivalent intrinsic _mm256_maskz_cvt_roundps_ph instead
since clang has always defined it as 3-argument.
https://trac.macports.org/ticket/67802

diff --git a/src/corelib/global/qfloat16.cpp b/src/corelib/global/qfloat16.cpp
--- src/corelib/global/qfloat16.cpp
+++ src/corelib/global/qfloat16.cpp
@@ -176,9 +176,9 @@
 void qFloatToFloat16_tail_avx256(quint16 *out, const float *in, qsizetype len) noexcept
 {
     __mmask16 mask = _bzhi_u32(-1, len);
     __m256 f32 = _mm256_maskz_loadu_ps(mask, in );
-    __m128i f16 = _mm256_maskz_cvtps_ph(mask, f32, _MM_FROUND_TO_NEAREST_INT);
+    __m128i f16 = _mm256_maskz_cvt_roundps_ph(mask, f32, _MM_FROUND_TO_NEAREST_INT);
     _mm_mask_storeu_epi16(out, mask, f16);
 };
 
 static QT_FUNCTION_TARGET(ARCH_SKYLAKE_AVX512)
